Use stride from package if installed
Index: b/bin/vmd.sh
===================================================================
--- a/bin/vmd.sh	2018-02-28 08:08:13.012738208 -0300
+++ b/bin/vmd.sh	2018-02-28 08:08:13.012738208 -0300
@@ -430,6 +430,14 @@
 # Stride -- used to generate cartoon representations etc.
 if [ -z "$STRIDE_BIN" ]
 then
+  if [ -x "/usr/bin/stride" ]
+  then
+    STRIDE_BIN="/usr/bin/stride"
+    export STRIDE_BIN
+  fi
+fi
+if [ -z "$STRIDE_BIN" ]
+then
   if [ -x "$MASTERVMDDIR/stride_$ARCH" ]
   then
     STRIDE_BIN="$VMDDIR/stride_$ARCH"
